Libraries/LibTLS/TLSv12.cpp

```diff
@@ -437,15 +437,14 @@ static void hmac_pseudorandom_function(Bytes output, ReadonlyBytes secret, u8 co
     HMACType hmac(secret);
     append_label_seed(hmac);
 
-    constexpr auto digest_size = hmac.digest_size();
-    u8 digest[digest_size];
-    auto digest_0 = Bytes { digest, digest_size };
+    auto digest_size = hmac.digest_size();
```

Comment:
why can't the digest size be constexpr here anymore?

Reply:
Because it gets the digest size as `m_inner_hasher->digest_size()` in `HMAC.h` and the `->` operator is not constexpr.

Reply:
Unfortunate but all right. The price we pay for more flexible interfaces.